for-posdealers/getting-started.md

```diff
@@ -253,10 +253,18 @@ Mit dem erfolgreichen Absetzen des Initial Operation Receipts ist die Inbetriebn
 
 ### Schritte nach erfolgreicher Inbetriebnahme
 
+#### Meldung der TSE-Kasse an das Finanzamt
+
 Mit der erfolgreichen Inbetriebnahme der Kasse mit TSE muss eine Meldung an das Finanzamt erfolgen. Derzeit steht dazu jedoch noch keine Möglichkeit der Meldung seitens der Finanzämter zur Verfügung. Wird von den Finanzämtern eine Möglichkeit zur Meldung zur Verfügung gestellt, wird durch fiskaltrust für Endverbraucher des fiskaltrust.Sorglos Paketes die Meldung an das Finanzamt automatisch eingereicht. Die bis dahin fälligen Meldungen werden ebenfalls automatisch nachgereicht, sobald die Möglichkeit seitens der Finanzverwaltung zur Verfügung steht. 
 
 Kunden ohne fiskaltrust.Sorglos Paket müssen die Meldungen selbständig beim zuständigen Finanzamt abgeben. Die dafür benötigten Informationen, wie z.B. Kassenseriennummer, TSE Seriennummer usw. sind im Portal verfügbar und können bei Bedarf vom Händler zur Verfügung gestellt werden.
```

Comment:
```suggestion
Kunden ohne fiskaltrust.Sorglos-Bundle müssen die Mitteilungen selbständig beim zuständigen Finanzamt abgeben. Die dafür benötigten Informationen, wie z.B. Kassenseriennummer, TSE-Seriennummer usw. sind im Portal verfügbar und können bei Bedarf auch vom KassenHändler zur Verfügung gestellt werden.
```

Reply:
Da das Produkt Finanzamtmeldung heisst, sprechen wir in der Produktbeschreibung konsistent von Meldungen. Den Rest habe ich übernommen!